### ISSUE TITLE
openapi 3.1.x support

### ISSUE THREAD

**Issue by hrvoj3e** [proohit/openapi-snippets#2]
Cool extension.
Will it support 3.1?

**Comment by proohit** [comment]
Yes, that's planned :). Are there any features that should be prioritized in your opinion?

As always, contributions are very much appreciated! Feel free to open a pull request if you have any changes :).

**Comment by hrvoj3e** [comment]
Priority would be to use existing 3.0 snippets when editing 3.1 spec. :)
If that is possible.

**Comment by proohit** [comment]
So I began working on supporting or rather migrating to 3.1.x. As it seems, they introduced a lot of breaking changes and also changed their used json schema. This introduces some challenges, because VS Code does not seem to support needed features of said json schema. So long, we're stuck here, as I don't want to validate the files myself but instead rely on VS Code. BTW according to their release notes they should support newer json schema drafts, but it still won't work. That's a bummer. I opened a SO question, maybe some other people have experienced a similar issue. If there are no answers, I will open an issue on VS Code.
SO: https://stackoverflow.com/questions/72958304/meta-schema-features-in-vs-code-extension-warning-in-version-1-68-0

**Comment by proohit** [comment]
Issue in VS Code: https://github.com/microsoft/vscode/issues/155379
